svm-rent-collector/src/svm_rent_collector.rs

```diff
@@ -9,8 +9,8 @@ use {
         rent::{Rent, RentDue},
         rent_collector::CollectedInfo,
         transaction::{Result, TransactionError},
-        transaction_context::{IndexOfAccount, TransactionContext},
     },
+    solana_transaction_context::{IndexOfAccount, TransactionContext},
```

Comment:
We don't want the SVM to be dependent upon the validator, so maybe the Solana-transaction context belongs in the SVM? cc @pgarg66

Reply:
Well more specifically it belongs in the program-runtime. But there are some annoying usesites (such as RPC) which we have to untangle. So the un-crating will happen in a separate PR.

Reply:
We don't want SVM to depend on runtime either.

Reply:
`program-runtime` should be ok, since it's part of SVM.